### ISSUE TITLE
Session timeout settings

### ISSUE THREAD

**Issue by tomas-knap** [mff-uk/odcs#326]
I used spring security for handling session, and the default timeout should be 30 minutes. Generally in Spring Security this can be changed on HTTP session object. However vaadin tries to hide HTTP communication, so I am not sure how to change it. It would require more research. I believe 30 minutes is OK for timeout, but in future it would probably be better to add session timeout setting to configuration.

**Comment by tomas-knap** [comment]
Still relevant?

**Comment by janvojt** [comment]
Yes, you can set session timeout in web.xml. Otherwise, if you wish to read it from config.properties, you probably need to obtain session from RequestHolder, and then use method setMaxInactiveInterval().
